app/pubspec.lock

```diff
@@ -444,13 +437,6 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.1.0-nullsafety.1"
-  stream_transform:
```

Comment:
I thought @MummenRider was using this?

Reply:
Nope, I don't remember adding this. If you're talking about stream or streambuilder, then that is different
> I thought @MummenRider was using this?

Reply:
Yup, flutter pub upgrade was doing something. I deleted pubspec.lock and ran flutter pub get again, and this is still gone. Likely some funkiness with rebases here.